[PATCH] e2fsprogs: fix tests/f_extent_oobounds

tests/f_extent_oobounds runs debugfs from the system, not
from the source tree, and if the system's debugfs doesn't
have the extent_open command it fails silently.

Use $DEBUGFS to get the right executable.

Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

diff --git a/tests/f_extent_oobounds/script b/tests/f_extent_oobounds/script
index 31ac6c9..b1f0e1a 100644
--- a/tests/f_extent_oobounds/script
+++ b/tests/f_extent_oobounds/script
@@ -5,7 +5,7 @@ TEST_DATA="$test_name.tmp"
 
 dd if=/dev/zero of=$TMPFILE bs=1k count=256 > /dev/null 2>&1
 mke2fs -Ft ext4 $TMPFILE > /dev/null 2>&1
-debugfs -w $TMPFILE << EOF  > /dev/null 2>&1
+$DEBUGFS -w $TMPFILE << EOF  > /dev/null 2>&1
 write /dev/null testfile
 extent_open testfile
   insert_node 0 15 100

